include/nbl/video/COpenGL_Swapchain.h

```diff
@@ -0,0 +1,64 @@
+#ifndef __NBL_VIDEO_C_OPENGL__SWAPCHAIN_H_INCLUDED__
+#define __NBL_VIDEO_C_OPENGL__SWAPCHAIN_H_INCLUDED__
+
+#include "nbl/system/IThreadHandler.h"
+#include "nbl/video/ISwapchain.h"
+
+namespace nbl::video
+{
+
+class IOpenGL_LogicalDevice;
+class COpenGLFeatureMap;
+class COpenGLSync;
+class COpenGLSemaphore;
```

Comment:
do you need these forward declarations here, can you not just include them in the CPP file?